examples/v3.0/petstore-expanded.yaml

```diff
@@ -79,7 +79,8 @@ paths:
                 $ref: '#/components/schemas/Error'
   /pets/{id}:
     get:
-      description: Returns a user based on a single ID, if the user does not have access to the pet
+      description: Returns a pet based on a single ID
+      #Description seems quite right before
```

Comment:
I think it would be confusing to leave a comment such as this here.

Reply:
OK Thank you very much.  Obviously I missed something, Anyhow thank you very much for the really helpful samples.